src/megabot-internals/controllers/inquirer.js

```diff
@@ -11,6 +11,11 @@ module.exports = {
    * @returns {Promise<Object>} - Database response
    */
   createChatvote: (msg, id, reportable = true) => {
+    const existing = db.findSync('questions', {
+      zd_id: parseInt(id)
+    })
+    if (existing && (Date.now() - existing.meta.created) < 30000) return // If there's a document created less than 30 seconds ago with the same ID, we don't want to create a chatvote
```

Comment:
We still want to display the voting reactions

```suggestion
    if (existing && (Date.now() - existing.meta.created) < 30000) reportable = false // If there's a document created less than 30 seconds ago with the same ID, we don't want to create a chatvote
```